curiouslearning/FeedTheMonsterJS
# fix: Next, Retry button issues and double blick stacking issue when pressing the button (FM-328)

# Changes
- fix level end buttons and button click audio

# How to test
- Open the start screen of FTM.
- Tap or click any part of the start screen to proceed to the next screen
- Select any level
- Play the game until the end
- In level end scene
- Press any button
- Make sure button click audio only play once
- Make sure it proceeds to next level
- Make sure Retry button is working
- Make sure Map button is working

Ref: [FM-328](https://curiouslearning.atlassian.net/browse/FM-328)


[FM-328]: https://curiouslearning.atlassian.net/browse/FM-328?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
